code/espurna/config/arduino.h

```diff
@@ -96,6 +96,7 @@
 //#define JINVOO_VALVE_SM_AW713
 //#define JORGEGARCIA_WIFI_RELAYS
 //#define KMC_70011
+#define KOGAN_SMARTER_HOME_PLUG_W_POW
```

Comment:
Please comment this out.
*I wonder how CI test can catch this too. It already overrides device flag, so this does nothing there.*